cmd/fabric8-jenkins-proxy/main.go

```diff
@@ -111,6 +112,24 @@ func start(config configuration.Configuration, tenant *clients.Tenant, wit clien
 	wg.Wait()
 }
 
+func listenAndServeHelper(srv *http.Server, cancel context.CancelFunc) {
+	IsSslEnabledStr, _ := os.LookupEnv("JC_ENABLE_SSL")
+	bSslEnabled, parseErr := strconv.ParseBool(IsSslEnabledStr)
+	if parseErr == nil {
```

Comment:
if `err != nil` is generally called right after the call that returns the error, it's probably not idiomatic to check `if err == nil`.